src/components/auth/form.tsx

```diff
@@ -22,7 +22,7 @@ const AuthForm = ({
       action={action}
       method="post"
       className="mt-4"
-      onSubmit={async (e) => {
+      onSubmit={async e => {
```

Comment:
This is the auto-fix? How about we use with bracket all the time?

NOTE: This is just my preference.

Reply:
✅ Done